src/tree/tree.scss

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+$iui-child-node-spacing: calc(36px * (var(--level, 0)));
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+}
+
+@mixin iui-tree-node {
+  @include iui-focus;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding: $iui-xxs;
+  cursor: pointer;
+  box-sizing: border-box;
+  user-select: none;
+  margin-right: $iui-sm;
+  //min-height: $iui-component-height-small + (2 * $iui-xxs); //button height + padding
+  min-height: $iui-baseline * 3;
+
+  > .iui-icon {
+    @include iui-icons-default;
+    margin-right: $iui-s;
+  }
+
+  > .iui-button {
+    margin-right: $iui-s;
+    margin-left: $iui-s;
+
+    &:first-child {
+      margin-left: $iui-child-node-spacing;
+    }
+
+    > .iui-icon {
+      @media (prefers-reduced-motion: no-preference) {
+        transition: transform $iui-speed-fast ease-out;
+      }
+    }
+  }
+
+  > .iui-checkbox {
+    margin-left: $iui-s;
+    margin-right: $iui-child-node-spacing;
+  }
+
+  .iui-node-label {
+    min-width: 0;
+
+    &:only-child {
+      margin-left: $iui-child-node-spacing;
+    }
+  }
+
+  .iui-title,
+  .iui-caption {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .iui-title {
+    font-size: $iui-font-size;
+  }
+
+  .iui-caption {
+    font-size: $iui-font-size-small;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  &.iui-active {
+    &,
+    &:hover {
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+        outline: $iui-active-outline;
+        outline-offset: -1px;
+      }
+    }
+
+    &:focus,
+    &.iui-focused {
+      outline-width: $iui-xxs;
+      outline-offset: -2px;
+    }
+
+    &:focus:not(:focus-visible) {
+      outline: $iui-active-outline;
+      outline-offset: -1px;
+    }
+  }
+
+  &.iui-disabled {
+    &,
+    &:hover {
+      cursor: not-allowed;
+      outline: none;
+      @include themed {
+        background-color: t(iui-color-background-1);
+        color: t(iui-text-color-placeholder);
```

Comment:
In the tree we support a secondary line of text that uses the muted text color.  Because of this, our disabled row needs to use an even lighter text color, placeholder:
![Screen Shot 2021-11-22 at 11 13 53 AM](https://user-images.githubusercontent.com/849817/142896274-631bd6ff-9f60-40b1-a9cc-8adf68009ba0.png)

Our table does _not_ support a secondary line of text that uses the muted color.  We should update the table's disabled rows text color to align with the tree.

Reply:
That will be an accessibility concern, especially in light theme.

In Table, we make the disabled rows have italic text. So it might be fine to leave it like that. But italic might not look good in Tree.

How often do we expect the disabled node pattern to be used?

Reply:
The italics within the table is also slightly incorrect on my behalf as well.  My intent was for a disabled row to look like this without italics:
![Screen Shot 2021-11-22 at 12 38 51 PM](https://user-images.githubusercontent.com/849817/142909043-c746e329-5888-41b2-962e-18d2f00672df.png)
 
The italics was meant to symbolize that the file is processing / uploading:
![Screen Shot 2021-11-22 at 12 39 40 PM](https://user-images.githubusercontent.com/849817/142909140-2db2cdc2-107e-4550-ae8e-07275725fd9f.png)

I think this could be accomplished by creating an additional class like `.iui-processing` to add the italics & decouple the italics from the `.iui-disabled`.


And to answer you last question, I have no gauge on how often we expect tree nodes to be disabled.